pkg/audit/audit.go

```diff
@@ -38,10 +38,21 @@ func (m *LabelMatcher) Match(labels *BackendLabels) bool {
 	return false
 }
 
+// Sequence is used to help backend implement audit.Backend
```

Comment:
Where does the `Sequence` init?

Reply:
creates and inits `Sequence ` when  creating `Backend`